src/test/java/com/google/javascript/clutz/partial/missing_base.js

```diff
@@ -0,0 +1,33 @@
+goog.module('mising.extend');
+
+//!! Using class syntax requires default externs but the test runs faster
+//!! without them. Just use @constructor instead.
+
+/**
+ * @constructor
+ * @extends {direct.ref.A}
+ */
+function B() {
+
+}
+
+/**
+ * @constructor
+ * @extends {direct.ref.AT<string, number>}
+ */
+function BT() {
```

Comment:
The naming here is not super obvious for me – maybe `MissingParentClassTemplated` would be clearer than `BT`?

Reply:
renamed to BTemplated